clients/intro_course_developer_component/src/introCourse/network/mutations/createAppleAccount.ts

```
@@ -0,0 +1,25 @@
+      `intro-course/api/course_phase/${coursePhaseID}/developer_account/invite/${courseParticipationID}`,
+      {},
+      {
+        headers: {
+          'Content-Type': 'application/json-path+json',
+        },
+      },
+    )
+    return response.data
+  } catch (err: any) {
+    console.error(err)
+    if (err.response && err.response.data && err.response.data.error) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)